tests/ToolInfoTest.php

```diff
@@ -21,13 +21,87 @@
  */
 final class ToolInfoTest extends \PHPUnit_Framework_TestCase
 {
+    const PHPUNIT_BIN = '/../vendor/bin/phpunit';
+
+    protected function setUp()
+    {
+        foreach ($this->provideScriptName() as $testArguments) {
+            $symlink = current($testArguments);
+            if ($symlink === self::PHPUNIT_BIN or is_link(__DIR__.$symlink)) {
```

Comment:
Will change. As of yet I'm trying to figure out how to run all on Windows.

OT: I cannot understand the love of the community for `||` and `&&`, code should be readable as fast as possible. Words will always be faster than symbols.